src/js/edu-benefits/1990-rjsf/helpers.jsx

```diff
@@ -23,33 +23,24 @@ export const benefitsEligibilityBox = (
   </div>
 );
 
-export const benefitsLabels = {
-  chapter33:
-    (<p>
-      Post-9/11 GI Bill (Chapter 33)
-      <br/>
-      <a href="/education/gi-bill/post-9-11/" target="_blank">Learn more</a>
-    </p>),
-  chapter30: (
-    <p>
-      Montgomery GI Bill (MGIB-AD, Chapter 30)
-      <br/>
-      <a href="/education/gi-bill/montgomery-active-duty/" target="_blank">Learn more</a>
-    </p>
-  ),
-  chapter1606: (
-    <p>
-      Montgomery GI Bill Selected Reserve (MGIB-SR, Chapter 1606)
-      <br/>
-      <a href="/education/gi-bill/montgomery-selected-reserve/" target="_blank">Learn more</a>
-    </p>
-  ),
-  chapter32: (
-    <p>
-      Post-Vietnam Era Veterans’ Educational Assistance Program <br/>
-      (VEAP, Chapter 32)
-      <br/>
-      <a href="/education/other-educational-assistance-programs/veap/" target="_blank">Learn more</a>
-    </p>
-  )
+export const benefitsRelinquishmentLabels = {
+  unknown: 'I’m only eligible for the Post-9/11 GI Bill',
+  chapter30: 'Montgomery GI Bill (MGIB-AD, Chapter 30)',
+  chapter1606: 'Montgomery GI Bill Selected Reserve (MGIB-SR, Chapter 1606)',
+  chapter1607: 'Reserve Educational Assistance Program (REAP, Chapter 1607)'
 };
+
+export const benefitsRelinquishmentWarning = (
+  <div>
+    <p>Because you chose to apply for your Post-9/11 benefit, you have to relinquish (give up) 1 other benefit you may be eligible for.</p>
+    <p><strong>Your decision is irrevocable</strong> (you can’t change your mind).</p>
+    <br/>
+  </div>
+);
+
+export const benefitsRelinquishedDescription = (
+  <span>
+    <br/>
+    If you have questions or don’t understand the choice, talk to a specialist at 1-888-442-4551 (1-888-GI-BILL-1) from 8:00 a.m. - 7:00 p.m. ET Mon - Fri.
```

Comment:
That's an excellent question! To be honest, I just used chrome's dev tools to copy the element from the original 1990. I'll create a ticket to loop back around on this, but I'm not going to worry about it in this PR.